crates/config/src/config.rs

```diff
@@ -180,14 +181,25 @@ pub struct PscConfig {
     pub submitted_limit: u64,
     /// Minimum delay between blocks. Default is 8 seconds.
     pub block_interval_secs: u64,
+    pub min_fee_rate: u64,
+    pub max_fee_rate: u64,
+    pub fee_rate_pid: Option<Pid<f64>>,
```

Comment:
Do you have a tested config example for [fee_rate_pid](https://docs.rs/pid/latest/pid/struct.Pid.html)?
@sopium

Reply:
The sim program outputs PID params on exit, e.g.:

```toml
setpoint = 0.6
output_limit = 1.0
kp = -0.8
ki = -0.08
kd = 0.0
p_limit = 1.0
i_limit = 1.0
d_limit = 0.0
integral_term = -1.0
```